spec/helpers/article_helper_spec.rb

```diff
@@ -124,4 +124,37 @@
       expect(result).to eq '<p>This Journal</p>, 10(1)'
     end
   end
+
+  context '#transform_research_starter_text' do
+    it 'removes anid and title' do
```

Comment:
I'd write the subject of this context as:
```ruby
subject { helper.transform_research_starter_text(value: value) }
```
each subordinate context can set value:

```ruby
context "with EDS tags and regular HTML" do
  let(:value) { '<anid>abc</anid><title>123</title>' }
  it { is_expected.to eq '' }
end
```